sale.py

```diff
@@ -313,8 +314,14 @@ def create_using_ps_data(cls, order_record):
 
         sale, = cls.create([sale_data])
 
-        assert sale.total_amount == Decimal(str(
-            order_record.total_paid_tax_excl)), 'The order total do not match'
+        try:
+            assert sale.total_amount == Decimal(
+                str(order_record.total_paid_tax_excl)
+            ), 'The order total do not match'
+
+        except:
+            sale.prestashop_exception = True
```

Comment:
I think there should be a separate model called prestashop.exceptions and why sale is not processed that should be saved there like magento expection.